guides/advanced_guides/keys.md

```diff
@@ -1,145 +1,38 @@
 # Keys
 
-To communicate with MeiliSearch's `RESTful API` most of the routes require an `API key`.
+In MeiliSearch there are three types of keys:
 
-The following information defines a key:
-* A **[Key](/guides/advanced_guides/keys.md#key)** generated by MeiliSearch
-* A **[Description](/guides/advanced_guides/keys.md#description)**
-* An **[ACL:](/guides/advanced_guides/keys.md#acl)** list of permissions
-* An **[Indexes list](/guides/advanced_guides/keys.md#indexes)**  with wildcard on which the ACL's are applied
-* A **[Revoked](/guides/advanced_guides/keys.md#revoked)** updatable boolean to revoke a key <Badge text="soon" type="warn"/>
-* An **[Expires_at](/guides/advanced_guides/keys.md#expires-at)** timestamp that indicated the expire date of the token
+- **Master** key: access to all routes.
+- **Private** key: access to all routes except the `/keys` routes.
+- **Public** key: only access:
+    - `GET /indexes/search`
+    - `GET /indexes/:index_uid/search`
+    - `GET /indexes/:index_uid/documents`
+    - `GET /indexes/:index_uid/documents/:doc_id`
 
-A key is passed to MeiliSearch using the header
-` X-Meili-Api-Key: myApiKey `
+When a master key is given, MeiliSearch will have one master key and generate one private key and one public key. **You can not create additionals keys**.
```

Comment:
```suggestion
When a master key is given to MeiliSearch, the engine will also generate one private, and one public key. **You cannot create additional keys**.
```

Please in the future, could you use "cannot" instead of "can not", IIRC that's your 42 friend who told us about that :)